packages/global/components/document.marko

```diff
@@ -10,6 +10,7 @@ $ const {
 } = out.global;
 
 $ const continueLable = i18n("Sign Me Up!");
```

Comment:
`continueLable` is incorrectly spelled; is it wrong in the low-level component as well?